coalib/results/SourceRange.py

```diff
@@ -125,3 +125,7 @@ def __json__(self, use_relpath=False):
         if use_relpath:
             _dict['file'] = relpath(_dict['file'])
         return _dict
+
+    def __contains__(self, item):
+        file_condition = self.start.file == item.start.file
```

Comment:
is there any reason for having this in an extra variable and not anding directly? The variable name doesn't really add ckarity for me.